### PR TITLE
Create BLadd19339.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/add/BLadd19339.xml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+                        <msItem xml:id="ms_i1">
+                            <locus from="2r"/>
+                            <title type="complete" xml:lang="en" ref="LIT6169Letter">The Encyclical Letter of
+                              <persName ref="PRS6755MarkVII">Mark,
+                            the 108th patriarch of Alexandria,</persName> to the Church of Ethiopia</title>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">ዛቲ፡ ጦማረ፡ ሃይማኖት፡ ዘፈነዋ፡
+                              <persName ref="PRS6755MarkVII">ማርቆስ፡ ሊቀ፡ ጳጳሳት፡
+                            ዘእስክንድርያ።</persName> ኀበ፡ ኵሎሙ፡ አብያተ፡ ክርስቲያናት፡ ዘኢትዮጵያ፡ በእዴሁ፡ ላጳጳስነ፡ መቃርዮስ፡
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/add/BLadd19339.xml:52]: ለጳጳስነ

File: LondonBritishLibrary/add/BLadd19339.xml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+                            </handNote>
+
+                        </handDesc>
+
+
+                        <additions>
+                            <list>
+
+                                <item xml:id="a1">
+                                    <desc type="Unclear">Comment on the manuscript contect in
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/add/BLadd19339.xml:107]: content

File: LondonBritishLibrary/add/BLadd19339.xml [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                          ሐዲስ፡ ሐዋርያ። ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስሌነ፡ አሜን። ። ። በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ።
+                        ይቤ፡ ማርቆስ፡ ሊቀ፡ ጳጳሳት። ዘውእቱ፡ ሳምናይ፡ እም ፻ወ፮ ዘተሠመይሙ፡ እማርቆስ፡ ወንጌላዊ።</incipit>
+                        <colophon xml:id="coloph1" xml:lang="gez"><locus from="20r"></locus>
+                        ወተጽሕፈ፡ ዝንቱ፡ መጽሐፍ፡ <date when-custom="1525" calendar="diocletian" when="1808">በ፲ወ፭፻፳ወ፭ ዓመተ፡
+                        ሰማዕታት፡ ንጹሓን።</date> በዘመነ፡ ማቴዎስ፡ አመ፡ ፱ ለወርኃ፡ ጥቅምት፡ ተወጥነ።
+                        ወአመ፡ ፲ወ፯ ተፈጸመ። ወዕላዎ፡ እምልሳነ፡ ዓረቢ፡ ለግዕዝ፤ <persName ref="PRS13140Giyorgis">ጊዮርጊስ፡ ግብጻዊ፡</persName> በሥምረቱ፡
+                        <persName ref="PRS9978Waldase">ለርእሰ፡ መኳንንት፡ ወልደ፡ ሥላሴ፡</persName>
+                        ወበመንግሥቱ፡ <persName ref="PRS3692egwalas">ለንጉሥነ፡ እጓለ፡ ጽዮን፡</persName> በቅድመ፡
+                        ኵሎሙ፡ ሊቃውንት፡ እለ፡ ተሰዱ፡ በኢትዮጵያ፤ በእንተ፡ ዛቲ፡
+                        ርትዕት፡ ሃይማኖተ፡ እስክንድያ፤ መንበረ፡ ስብከቱ፡ ለማርቆስ፡ ሐዋርያ።<gap reason="ellipsis"></gap>
```
Comment (thea-m) [inline LondonBritishLibrary/add/BLadd19339.xml:62]: እስክንድርያ?

File: LondonBritishLibrary/add/BLadd19339.xml [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+                    </physDesc>
+
+                    <history>
+                        <origin>
+                            <origDate notBefore="1808" notAfter="1867" evidence="lettering"/>
+                        </origin>
+                        <provenance>The manuscript has been purchased by
+                          <placeName ref="INS00001BL"></placeName> in <date>1847-1867</date>
+                          from the Library of the Earl of <placeName ref="wd:Q1081960">Mountnorris</placeName>.</provenance>
```
Comment (thea-m) [inline LondonBritishLibrary/add/BLadd19339.xml:131]: you could also mark him up as a person (I think PRS1950Annesley)

File: LondonBritishLibrary/add/BLadd19339.xml [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+                            </list>
+                        </additions>
+
+
+                    </physDesc>
+
+                    <history>
+                        <origin>
+                            <origDate notBefore="1808" notAfter="1867" evidence="lettering"/>
```
Comment (thea-m) [inline LondonBritishLibrary/add/BLadd19339.xml:127]: the ante quem could probably be the date of Salt's return from Ethiopia? (That's how I would understand the English note, or do you think the letter was copied from the one given to Salt?)

**Review by thea-m — CHANGES_REQUESTED** [review]
Interesting!

**Review by thea-m — APPROVED** [review]

**Pull request merged**